policy/predicate/file.go

```diff
@@ -141,6 +143,47 @@ func (pred *OnlyChangedFiles) Trigger() common.Trigger {
 	return common.TriggerCommit
 }
 
+type NoChangedFiles struct {
+	Paths       []common.Regexp `yaml:"paths"`
+	IgnorePaths []common.Regexp `yaml:"ignore"`
+}
+
+var _ Predicate = &NoChangedFiles{}
+
+func (pred *NoChangedFiles) Evaluate(ctx context.Context, prctx pull.Context) (*common.PredicateResult, error) {
+	changedFiles := ChangedFiles{
+		Paths:       pred.Paths,
+		IgnorePaths: pred.IgnorePaths,
+	}
+
+	changedFilesPredicateResult, err := changedFiles.Evaluate(ctx, prctx)
+	if err != nil {
+		return nil, err
+	}
+
+	predicateResult := common.PredicateResult{
+		ValuePhrase:   "changed files",
+		SkipPhrase:    "",
+		Satisfied:     !changedFilesPredicateResult.Satisfied,
+		Values:        changedFilesPredicateResult.Values,
+		ConditionsMap: changedFilesPredicateResult.ConditionsMap,
+	}
+
+	if predicateResult.Satisfied {
+		predicateResult.ConditionPhrase = "do not match"
+		predicateResult.Description = "No changed files match the specified patterns"
+	} else {
+		predicateResult.ConditionPhrase = "should not match"
```

Comment:
I think this may be better as `"match"`. That way, the overall expression will read `This rule is skipped because the changed files [...] match the path patterns [...]`